### DISCUSSION TITLE
strikethrough does't work

### DISCUSSION THREAD

**Discussion by miraculusik** [nvim-neorg/neorg#755] [Q&A] [answered]
How can i show strikethrough properly?
`-strikethrough-` nothing happening
Note: everything else shows properly just strikethrough not showing

**Comment by NTBBloodbath** [comment]
Hey, please make sure that your colorscheme is setting up `@text.strike` tree-sitter highlighting group. You can check it out by using `:hi @text.strike`.

If it does then maybe try it using another terminal, some of them aren't capable of rendering stuff like undercurl text so _perhaps_ that could be the problem root.

Cheers!

**Reply by miraculusik** [reply]
> Hey, please make sure that your colorscheme is setting up `@text.strike` tree-sitter highlighting group. You can check it out by using `:hi @text.strike`.

this returning an error `E411: highlight group not found: @text.strike` how can i fix ?
My colorscheme is [kanagawa](https://github.com/rebelot/kanagawa.nvim)
I try most famous 3 different color schemes but they don't work

**Reply by max397574** [reply]
vim.api.nvim_set_hl(0,"@text.strike",{strikethrough=true})
put this somewhere in your config, after you load the colorscheme

**Reply by miraculusik** [reply]
> If it does then maybe try it using another terminal, some of them aren't capable of rendering stuff like undercurl text so _perhaps_ that could be the problem root.
> 
> Cheers!

I'm using alacritty terminal but i tried iterm2 for this but still it's not shows properly.

**Reply by miraculusik** [reply]
> vim.api.nvim_set_hl(0,"@text.strike",{strikethrough=true})

I added this to the config file but nothing changed

Now `hi @text.strike` returns this
`@text.strike   xxx cterm=strikethrough gui=strikethrough`

**Comment by mrossinek** [comment, marked as answer]
I suggest you try to test this outside of Neovim. Try the following directly in your terminal:
```
echo -e "\e[9mtest\e[0m"
```
If the word `test` is _not_ rendered with strikethrough, then you can debug this outside of Neovim. I suggest looking into your terminal, its settings, and maybe even your font.
If that word does render properly, I suggest you test various colorschemes, test vim (not neovim) and check whether it can render strikethrough properly, etc.

**Reply by miraculusik** [reply]
In my terminal `echo -e "\e[9mtest\e[0m"` it works.But in neovim doesn't render properly. I tried a lot of colorscheme but none of them works

**Reply by mrossinek** [reply]
And in what you posted [here](https://github.com/nvim-neorg/neorg/discussions/755#discussioncomment-5052354) the `xxx` gets rendered properly with strikethrough? If yes, then it is merely a colorscheme linking issue and you might need to link the necessary highlights manually (although that would surprise me).
If it does not render properly there, then this seems to be a deeper issue with your neovim and nothing specific to Neorg itself.

**Reply by miraculusik** [reply]
`xxx` just showing as it. i think you are right its related with my neovim config but i cant figure out :( 
i'm collecting all my notes in one place and i use neorg for this i hope i can find it soon. I will work on this problem thanks for helping.

**Comment by champignoom** [comment]
possibly related: https://github.com/neovim/neovim/issues/3436

**Reply by champignoom** [reply]
for me, `brew install zsh ncurses && echo /usr/local/bin/zsh |sudo tee -a /etc/shells && chsh -s /usr/local/bin/zsh && echo 'export TERM=tmux-256color' >>~/.zshrc` solves the problem.

not sure why but neither TERM=iterm2 nor TERM=xterm-256color works.

**Comment by d-r-a-b** [comment]
Take a look at https://gist.github.com/d-r-a-b/3af7083a018be15e6c1d1d2c5317e9c7

Specifically the sections on Terminals and the various troubleshooting steps. I specifically think that `:h checkhealth` is a good place to start, but all of it may be relevant.

**Comment by jkroes** [comment]
For anyone else like me, who checked the issues and the Wiki without finding a solution,  Neovide GUI on macos completely resolved rendering issues with neorg. I can now see strikethroughs, folding actually hides example blocks, `#color red` directives turn subsequent blocks red, etc. There were countless issues with rendering in the terminal.

**Comment by sibevin** [comment]
I encountered the same issue, but eventually discovered that the colorscheme plugin was outdated. After updating it, it displayed correctly.
